docs/datasets.md

```diff
@@ -55,3 +55,159 @@ To use `ROMS-Tools`, the user is required to download the following datasets:
+days = [31,29,31,30,31]
+for y in years:
+  for count,m in enumerate(months):
+    for d in range(1,days[count]+1):
+      if d < 10:
+        extra_str='0'
+      else:
+        extra_str=''
+      copernicusmarine.subset(
+        dataset_id="cmems_mod_glo_phy_my_0.083deg_P1D-m",
+        variables=["so", "thetao", "uo", "vo", "zos"],
+        minimum_longitude=-75,
+        maximum_longitude=-15,
+        minimum_latitude=2,
+        maximum_latitude=50,
+        start_datetime= y + "-" + m + "-" + extra_str + str(d) + "T00:00:00",
+        end_datetime= y + "-" + m + "-" + extra_str + str(d) + "T00:00:00",
+        minimum_depth=0.49402499198913574,
+        maximum_depth=5727.9,
+)
+```
+## ERA5
+
+### Setup
+
+Instructions on installation and how to set up account credentials, in the shell: [here](https://help.marine.copernicus.eu/en/articles/7970514-copernicus-marine-toolbox-installation)
+
+But, essentially do:
+
+- Make account at: https://cds.climate.copernicus.eu/
+- Login on the web
+- Setup the api: https://cds.climate.copernicus.eu/how-to-api
+    - Add this to `$HOME/.cdsapirc`:
+
+```bash
+url: https://cds.climate.copernicus.eu/api
+key: 77deaea8-7d41-4b8e-967f-90d6d1849c0e
+```
+
+- Make sure to have correct version: `$ pip install "cdsapi>=0.7.4"`
+- Sign the license agreement: https://cds.climate.copernicus.eu/profile?tab=licences
+
+### Download
+
+Run the download script below to extract data for *wind, dewpoint, temp, sst, precipitation, net solar radiation, surface downward thermal radiation*. Specify the years, months.
+
+```python
+# This script downloads the ERA5 files using cdsapi
+# The variables selected are those needed to create forcing files in roms-tools as of May 2025
+# before using this script, you must be in an environment with cdsapi installed
+
+import cdsapi
+import zipfile
+import shutil
+import os
+from tempfile import mkdtemp
+
+for yr in range(2017,2023):
+    for m in range(1,13):
+        dataset = "reanalysis-era5-single-levels"
+        request = {
+            "product_type": ["reanalysis"],
+            "variable": [
+                "10m_u_component_of_wind",
+                "10m_v_component_of_wind",
+                "2m_dewpoint_temperature",
+                "2m_temperature",
+                "sea_surface_temperature",
+                "total_precipitation",
+                "surface_net_solar_radiation",
+                "surface_thermal_radiation_downwards"
+            ],
+            "year": [str(yr)],
+            "month": [str(m).zfill(2)],
+            "day": [
+                "01", "02", "03",
+                "04", "05", "06",
+                "07", "08", "09",
+                "10", "11", "12",
+                "13", "14", "15",
+                "16", "17", "18",
+                "19", "20", "21",
+                "22", "23", "24",
+                "25", "26", "27",
+                "28", "29", "30",
+                "31"
+            ],
+            "time": [
```

Comment:
See [sample gist](https://gist.github.com/ankona/5e7471233810f4b8761191bf2716ff63#file-cds_helper-py-L143) if you want to avoid hardcoding times and days.